clean.cmd

```diff
@@ -0,0 +1,11 @@
+@echo off
+
+REM Copyright (c) .NET Foundation and contributors. All rights reserved.
+REM Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+if "%~1"=="" (

```

Comment:
Let's be consistent with other repos. I expect folks who run a thingie called `clean` will understand the risk :)